code/devextra.c

```diff
@@ -170,7 +170,7 @@ void clean_mud()
 	if (!nSQL.StartSQLServer(riftCore.Host.c_str(),
 		riftCore.Db.c_str(), riftCore.User.c_str(), riftCore.Pwd.c_str()))
 	{
-		RS.Bug("clean_mud: failed to establish a database connection.");
+		RS.Logger.Warn("clean_mud: failed to establish a database connection.");
```

Comment:
Do you feel like this might be better as an error?

Reply:
Fixed in latest commit.